src/components/AboutText/index.js

```diff
@@ -0,0 +1,11 @@
+import { about } from '../../data/about'
+
+export default function Preview() {
+  return (
+    <>
+      {about.map((about) => (
+        <p key={about.text}>{about.text}</p>
+      ))}
+    </>
```

Comment:
You have only one text contained in the about array. I think map() is unnecessary here also the fragments are unnecessary I think

Reply:
here i'm also just no longer sure whether the fragments are necessary🤔💎